<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDipl</class>
 <widget class="QWidget" name="FormDipl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Что писать?</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Выравнивание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Шрифт</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Растояние от левого края, мм</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Растоние от верхнего края, мм</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Длина строки, мм</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="text">
         <string>Добавить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Удалить строку</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBoxWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100500</number>
        </property>
        <property name="value">
         <number>210</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="spinBoxHeight">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100500</number>
        </property>
        <property name="value">
         <number>297</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ширина листа</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Высота листа</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>* При печати PDF укажите 100% масштаб (без полей, отступов и т.д.)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonPrint">
        <property name="text">
         <string>Печать...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>** Если текст будет вылезать за ширину строки, то его размер автоматически уменьшится, чтобы полностью поместиться на строке</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
